src/Samples/DIPS.Xamarin.UI.Samples/Controls/Toast/ToastPage.xaml.cs

```diff
@@ -0,0 +1,142 @@
+using System.Threading.Tasks;
+using System.Windows.Input;
+using DIPS.Xamarin.UI.Controls.Toast;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+using Color = System.Drawing.Color;
+using ToastControl = DIPS.Xamarin.UI.Controls.Toast.Toast;
+
+namespace DIPS.Xamarin.UI.Samples.Controls.Toast
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class ToastPage : ContentPage
+    {
+        private string m_pageTitle;
+        private ICommand m_showToastCommand;
+        private ICommand m_showToastCommand2;
+        private ICommand m_showToastCommand3;
+
+        public ToastPage()
+        {
+            InitializeComponent();
+
+            PageTitle = "Hello, World!";
+
+            ShowToastCommand = new Command(parameter =>
+            {
+                ToastControl.Current.ToastLayout = LayoutOneMoonFourMars;
+                ToastControl.Current.HideToastIn = 5;
+                ToastControl.Current.ToastAction = async () =>
+                {
+                    PageTitle = "Hello, Mercury!";
+                    await Task.Delay(1500);
+                    PageTitle = "Hello, World!";
+                };
+                ToastControl.Current.Text = parameter.ToString();
+
+                ToastControl.Current.DisplayToast();
+            });
+
+            ShowToastCommand2 = new Command(parameter =>
+            {
+                ToastControl.Current.ToastLayout = LayoutTwoVenus;
+                ToastControl.Current.HideToastIn = 0;
+                ToastControl.Current.ToastAction = null;
+                ToastControl.Current.Text = parameter.ToString();
+
+                ToastControl.Current.DisplayToast();
```

Comment:
Maybe it would be better if layout was a nullable param in displaytoast instead of a mutable property?

Reply:
moved to a similar approach